src/irr/video/COpenGLSpecializedShader.cpp

```diff
@@ -88,7 +88,7 @@ static GLenum ESS2GLenum(asset::E_SHADER_STAGE _stage)
 
 }//namesapce impl
 
-COpenGLSpecializedShader::COpenGLSpecializedShader(const video::IVideoDriver* _driver, const asset::ICPUBuffer* _spirv, const asset::ISpecializationInfo* _specInfo) :
+COpenGLSpecializedShader::COpenGLSpecializedShader(const video::IVideoDriver* _driver, const asset::ICPUBuffer* _spirv, const asset::ISpecializationInfo* _specInfo, const asset::CIntrospectionData* _introspection) :
```

Comment:
pass the version instead of whole driver... also should the Version be a static var of `COpenGLExtensionHandler` ?

Reply:
k
>also should the Version be a static var of COpenGLExtensionHandler
I don't like global vars so ill make a parameter for it anyway